test/bats/05_auth.bats

```diff
@@ -25,6 +25,10 @@ setup () {
 }
 
 @test "Authenticated GET /models" {
+    if [[ ! -d /var/lib/nethserver ]] ; then
+        # Not NethServer, skip authentication test
```

Comment:
Not NethServer: let's move tests for upstream code to the upstream repository ;)